Microsoft.Toolkit/Extensions/StringExtensions.cs

```diff
@@ -95,6 +95,31 @@ public static bool IsCharacterString(this string str)
             return Regex.IsMatch(str, CharactersRegex);
         }
 
+        /// <summary>
+        /// Finds the number of occurrences of a substring exists inside of a string.
+        /// </summary>
+        /// <param name="searchString">String to search through</param>
+        /// <param name="toFind">Substring to find</param>
+        /// <returns>Number of times the substring occurs</returns>
+        public static int Occurrences(this string searchString, string toFind)
```

Comment:
Not sure this method is necessary.

Reply:
Also, I'm not sure it does what it's being documented. Why are you checking every character in toFind with every character in searchString?

Reply:
Yeah, the current logic only works in the single character case (which is how it's used in the PR).  If we want a function like this, we should add unit tests and make sure it works for the other scenarios.  Though it'd probably be easier/faster to use the built-in `IndexOf` or `Split` methods to do this, no?